src/TextInput.tsx

```diff
@@ -144,4 +161,5 @@ TextInput.propTypes = {
 
 TextInput.displayName = 'TextInput'
 
+export type TextInputProps = ComponentProps<typeof TextInput>
 export default TextInput
```

Comment:
I'm assuming wrapper props doesn't need it's own export because it's something that's used internally here that would not need to be exposed outside this component?

Reply:
Yup 👍 No need to expose implementation details